docs/content/documentation/systems/Materials/tensor_mechanics/ComputeInterfaceStress.md

```diff
@@ -0,0 +1,58 @@
+# ComputeInterfaceStress
+!description /Materials/ComputeInterfaceStress
+
+The resulting stress tensor has the property of having one Eigenvector in the
+direction of the order parameter gradient, with an Eigenvalue of zero, and two
+Eigenvectors perpendicular to that direction, with an Eigenvalue $\sigma_i\cdot|\nabla\eta|$,
+where $\sigma$ (`stress`) is a supplied parameter and $\eta$ (`v`) is a given order
```

Comment:
I suggest clarifying that here, sigma is a scalar since most people are going to think of it as a tensor. Also in the previous equation the parameter is called sigma_i, I would make it consistent between the equation and the text.